src/govuk/components/header/header.yaml

```
@@ -2,23 +2,23 @@ params:
 - name: homepageUrl
   type: string
   required: false
-  description: The url of the homepage. Defaults to /
+  description: The URL of the homepage. Defaults to /
```

Do you think we should mark the default `/` up as code here?

```suggestion
  description: The URL of the homepage. Defaults to `/`
```

---

Great catch!